vitrina/orgs/models.py

```diff
@@ -164,6 +164,7 @@ class Representative(models.Model):
     content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
     object_id = models.PositiveIntegerField()
     content_object = GenericForeignKey("content_type", "object_id")
+    can_write = models.BooleanField(default=False)
```

Comment:
```suggestion
    can_write = models.BooleanField(default=False, verbose_name=_("Leidžiama keisti duomenis"))
```
Django model forms will automatically use `verbose_name` as modelform field label